sdk/tests/deqp/functional/gles3/es3fFboRenderTest.js

```diff
@@ -1476,7 +1471,7 @@ goog.scope(function() {
         /** @type {WebGLProgram} */
         var texFromFboShaderID = context.createProgram(texFromFboShader);
         /** @type {WebGLProgram} */
-        var flatShaderID = context.createProgram(texFromFboShader);
+        var flatShaderID = context.createProgram(flatShader);
```

Comment:
C++ code:
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/functional/es3fFboRenderTest.cpp#1123

@zhenyao  the new code fixed your comments and added this new fix.